ansible/roles/reboot/tasks/main.yml

```diff
@@ -0,0 +1,27 @@
+---
+# .. vim: foldmarker=[[[,]]]:foldmethod=marker
+
+# Copyright (C) 2022 Julien Lecomte <julien@lecomte.at>
+# Copyright (C) 2022 DebOps <https://debops.org/>
+# .. SPDX-License-Identifier: GPL-3.0-only
+
+- name: Check if reboot is required
+  ansible.builtin.stat:
+    path: '/var/run/reboot-required'
+    get_checksum: no
+  register: reboot_required
```

Comment:
Use namespacing in role variables. This variable should be named `reboot__register_required`.